src/tribler-gui/tribler_gui/widgets/torrentfiletreewidget.py

```diff
@@ -107,12 +107,12 @@ def subtree_progress_update(self, updates, force_update=False, draw_progress_bar
                 self.progress_bytes = self.progress_bytes - old_bytes + new_bytes
 
         if child_changed or force_update:
-            if self.file_size and self.progress_bytes:
+            if self.file_size is not None and self.progress_bytes is not None:
```

Comment:
**question:** Is it safe regarding to possible `ZeroDivisionError`? Can `self.file_size` be zero here?

Reply:
Good point, it is safer to leave the check for non-zero value just in case :+1: